src/lib/containers/drug_upload_tool/DrugUploadForm.tsx

```diff
@@ -97,34 +102,28 @@ export default class DrugUploadForm extends React.Component<
       currentStep: steps[0],
       steps: steps,
       previousStepIds: [],
-      formData: props.formData || {},
-      formSchema: props.schema,
-
+      formData: props.formData,
       doShowErrors: false
     };
   }
 
   componentDidMount() {
```

Comment:
it looks like componentDidMount is setting isLoading to false -- is this intentional? theres no computation being made so this seems unnecessary